fdm-app/app/components/blocks/atlas-fields/cultivation-history.tsx

```
@@ -0,0 +1,131 @@
+            <CardHeader>
+                <CardTitle>Gewashistorie</CardTitle>
+                <CardDescription>
+                    De gewassen van de afgelopen jaren op dit perceel volgens
+                    Basisregistratie Gewaspercelen.
+                </CardDescription>
+            </CardHeader>
+            <CardContent className="pt-6 text-sm">
+                <div className="relative pl-1">
+                    {[...Array(3)].map((_, index) => (
+                        <div
+                            key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)